<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aubp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="showBread">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>21</y>
      <width>341</width>
      <height>145</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="whiteL">
       <property name="text">
        <string>White Count:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="whiteIN"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="wheatL">
       <property name="text">
        <string>Wheat Count:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="wheatIN"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="catPrompt">
       <property name="text">
        <string>Catering: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mixed(FEET)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom(STICKS)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="whiteCL">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Catering White</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="whiteCIN">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="wheatCL">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Catering Wheat</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="wheatCIN">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Debug">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuReset">
     <property name="title">
      <string>Reset</string>
     </property>
     <addaction name="actionLog"/>
     <addaction name="actionData"/>
    </widget>
    <addaction name="actionRevert_stcks_Left_to_last_value"/>
    <addaction name="actionShow_Log"/>
    <addaction name="menuReset"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit_values"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_me_please"/>
    <addaction name="actionContact_me_about_issues"/>
    <addaction name="queryUpdate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionHelp_me_please">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="actionContact_me_about_issues">
   <property name="text">
    <string>Feedback/Issues?</string>
   </property>
  </action>
  <action name="actionEdit_values">
   <property name="text">
    <string>Values</string>
   </property>
  </action>
  <action name="actionRevert_stcks_Left_to_last_value">
   <property name="text">
    <string>&amp;Revert Sticks Left To Previous</string>
   </property>
  </action>
  <action name="actionReset_Data">
   <property name="text">
    <string>Reset Data</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionData">
   <property name="text">
    <string>Data</string>
   </property>
  </action>
  <action name="actionShow_Log">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="queryUpdate">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
